src/main/java/com/epam/esm/GiftShop.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.esm;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
+
+@SpringBootApplication
+public class GiftShop extends SpringBootServletInitializer {
```

Comment:
Do we really need SpringBootServletInitializer to run the app?

Reply:
Fixed